Cython/Compiler/Parsing.py

```diff
@@ -1760,11 +1758,11 @@ def p_from_import_statement(s, first_statement = 0):
     # s.sy == 'from'
     pos = s.position()
     s.next()
-    if s.sy == '.':
+    if s.sy in ('.', '...'):
```

Comment:
I tried to come up with alternatives to this, but it feels like generally parsing sequences of dots helps no-one. Keeping only these two (i.e. making `...` special and nothing else) seems reasonable.